rclpy/rclpy/node.py

```diff
@@ -1949,3 +1950,22 @@ def get_subscriptions_info_by_topic(
             topic_name,
             no_mangle,
             _rclpy.rclpy_get_subscriptions_info_by_topic)
+
+    def wait_for_node(
+        self,
+        node_name: str,
```

Comment:
how about adding namespace as well which could be empty if user application does not use.

Reply:
I've updated the documentation to clarify this should by a fully qualified name (ie. including the namespace).